csrankings-0.csv

```diff
@@ -1830,6 +1831,7 @@ Ashutosh Bhatia,BITS Pilani,http://www.bits-pilani.ac.in/pilani/ashutoshbhatia/p
 Ashutosh Dhekne,Georgia Institute of Technology,https://www.cc.gatech.edu/~dhekne,gGB3lA4AAAAJ
 Ashutosh Gupta 0001,IIT Bombay,https://www.cse.iitb.ac.in/~akg,NOSCHOLARPAGE
 Ashutosh Kumar Gupta 0001,IIT Bombay,https://www.cse.iitb.ac.in/~akg,NOSCHOLARPAGE
+Ashutosh Modi, IIT Kanpur, https://ashutosh-modi.github.io/

```

Comment:
Similar problems (spaces, no Google Scholar ID).

Reply:
Hi,

Thank you for your response. I have created a new pull request incorporating the suggestions in https://github.com/emeryberger/CSrankings/pull/3401.